app/views/compute_resources/form/_ovirt.html.erb

```diff
@@ -1,6 +1,7 @@
 <%= text_f f, :url, :size => "col-md-8", :help_block => _("e.g. https://ovirt.example.com/api"), :help_inline => documentation_button('5.2.7oVirt/RHEVNotes') %>
 <%= text_f f, :user, :help_block => _("e.g. admin@internal") %>
 <%= password_f f, :password, :unset => unset_password? %>
+<%= checkbox_f f, :filtered_api, :help_inline => _("If you don't have admin permissions you can use slower filtered API."), :label => _('User level API') %>
```

Comment:
I don't see a big difference between "User level API" vs "Administrator account". Before I change it once again, could we have some poll to get the final text please? Just note that administrator account switches the meaning of flag so I should probably rename the field in DB as well, I guess for developer filtered API is more meaningful though.